accounting/localizations/argentina.rst

```diff
@@ -278,6 +520,72 @@ expected.
 .. image:: media/argentina17.png
    :align: center
 
+The vendor bill number is structured in the same way that the invoices with the difference 
+that the document sequence is input by the user: “Document Prefix - Letter -  Document number".
+
+
+Validate Vendor Bill number in AFIP
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+As most companies have internal controls to verify that the vendor bill is related to an AFIP
+valid document,  an automatic validation can be set in :menuselection:`Accounting --> Settings --> 
+Argentinian Localization --> Validate document in the AFIP`, considering the following levels: 
+
+- **Not available:** The verification is not done (this is the default value).
+- **Available:**  The verification  is done, in case the number is not valid 
+  it only raises a warning but it allows you to post the vendor bill. 
+- **Required:** The verification  is done and it doesn't allow the user to 
+  post the vendor bill if the document number is not valid.
```

Comment:
```suggestion
- **Not available**: The verification is not done (this is the default value).
- **Available**: The verification  is done, in case the number is not valid 
  it only raises a warning but it allows you to post the vendor bill. 
- **Required**: The verification  is done and it doesn't allow the user to 
  post the vendor bill if the document number is not valid.
```
(for in GitHub commit)